downstairs/src/region.rs

```diff
@@ -407,14 +410,76 @@ impl Default for ExtentMeta {
 }
 
 /**
- * Produce a PathBuf that refers to the backing file for extent "number",
- * anchored under "dir".
+ * Produce the string name of the data file for a given extent number
  */
-fn extent_path<P: AsRef<Path>>(dir: P, number: u32) -> PathBuf {
+pub fn extent_file_name(number: u32, extension: Option<&str>) -> String {
```

Comment:
Should extension be an `Option<&str>` or might it make sense to have this be some other enum to limit the set of inputs we might see here?